backend/backend/settings.py

```diff
@@ -93,15 +95,16 @@
 
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
-
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'djongo',
```

Comment:
Isn't it supposed to be "django" instead of "djongo"?

Reply:
No, it should be djongo. Djongo is a Object Database Mapper framework for MongoDB and djongo